lib/api/core/janitor.js

```diff
@@ -102,7 +98,7 @@ class Janitor {
         return this.kuzzle.services.list.storageEngine.import(request)
           .then(res => {
             if (res.partialErrors && res.partialErrors.length > 0) {
-              throw new PartialError(`Some data was not imported for ${request.input.resource.index}/${request.input.resource.collection} (${res.partialErrors.length}/${res.items.length + res.partialErrors.length}).`, res.partialErrors);
+              errorsManager.throw('internal', 'janitor', 'import_data', res.partialErrors)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)